### DISCUSSION TITLE
RomWBW v3.4.0 Pending -- Last Call for Changes/Corrections

### DISCUSSION THREAD

**Discussion by wwarthen** [wwarthen/RomWBW#377] [General]
Hi all,

I am in the process of finalizing a new stable release of RomWBW (v3.4.0).  I intend to publish this release by the new year.  If anyone has any issues or contributions that you want considered for this release, please contact me ASAP or reply in this discussion.

**NOTE:** I will be changing the branch conventions when this release is published.  Currently, the master branch is the latest stable version of the code and the dev branch is used for development.  I will be changing to using master for development and will create a branch called stable for the latest stable release.  This is more consistent with typical repository conventions.  I will be removing the dev branch at some point.

Thanks!

Wayne

**Comment by feilipu** [comment]
As a suggestion, unless you intend to take on the extra workload of maintaining a stable version, you don't need to create a stable branch. Just do Releases off the main or master branch.

**Reply by wwarthen** [reply]
An excellent point.  I can always create a stable branch as needed from the release tag.

**Reply by feilipu** [reply]
Just to add a thought for consideration, with [z88dk/z88dk](https://github.com/z88dk/z88dk) any major improvements or new features are usually done in feature branches, forked from the main branch, that are integrated into main branch via a PR once they are completed and tested working.

Interested people can also test by switching to the repository specific feature branch. 

That keeps the main branch clean and building properly, but allows several developers to work at their own speed on new features or issues.

It might be a useful paradigm for RomWBW too, because you could start new things on branches and not be compelled to have them working fully before fixing something else (small) on the main branch.

**Reply by wwarthen** [reply]
Also a good suggestion.  At the moment, I am the only one with permissions to change the repository.  Forking and Pull Requests have been working OK.  I will think about this though.

Thanks you.

**Comment by wwarthen** [comment]
An initial Release Candidate for v3.4 has been published as a Development Snapshot.  It is available on the [Releases](https://github.com/wwarthen/RomWBW/releases) page or via the dev branch.

If you design or distribute hardware that runs RomWBW, this would be a good release to use to confirm that v3.4 is working on your hardware.

Thanks,

Wayne

**Comment by wwarthen** [comment]
Closing this discussion since v3.4 has been released!
